lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,33 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
+    <%- edx.StringUtils.interpolate(
```

Comment:
Yes it would be available. its sibling template is using **edx.StringUtils.interpolate** (https://github.com/edx/edx-platform/blob/master/lms/templates/commerce/receipt.underscore#L100)

Reply:
You didnt changed the format string `gettext("You still need to visit the {display_name} website to complete the credit process.")`